### ISSUE TITLE
L2 does not work after calibrating the trigger

### ISSUE THREAD

**Issue by GGXXBB** [carpikes/ds4-tools#25]
hello.
I used this tool to calibrate the DS4 trigger, and after calibration, its L2 did not work, R2 was fine.

![QQ图片20240309142523](https://github.com/carpikes/ds4-tools/assets/50975096/6c493a36-2ad0-4ae2-a3ac-4e0915b8835c)
![QQ截图20240309142012](https://github.com/carpikes/ds4-tools/assets/50975096/0c1ce204-5ef9-4fa1-86cf-4cb294d193cb)

Thank you!

**Comment by carpikes** [comment]
Yup, the trigger calibration needs more rework. It's much more difficult than expected. Probably it behaves differently w.r.t the DS4 model.
